src/text/line.rs

```diff
@@ -218,6 +218,57 @@ impl<'a> Line<'a> {
         }
     }
 
+    /// Left-aligns this line of text.
+    ///
+    /// Convenience shortcut for `Line::alignment(Alignment::Left)`.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// # use ratatui::prelude::*;
+    /// let mut line = Line::from("Hi, what's up?");
+    /// assert_eq!(None, line.alignment);
+    /// assert_eq!(Some(Alignment::Left), line.left_aligned().alignment);
```

Comment:
This variable doesn't need to be mutable.

I generally prefer to avoid testing preconditions in tests like this (but I see that came from the alignment impl). Here I think we could reasonably get away with avoiding any assertions without too much loss of obviousness - what do you think?

```suggestion
    /// # use ratatui::prelude::*;
    /// let line = Line::from("Hi, what's up?").left_aligned();
```

Reply:
Sounds good, will change it also for the other two functions!